# Service that can be used to set the current EE used in MoveIt.
string ee_name
---
bool success
string message
